src/api/authApi.js

```diff
@@ -0,0 +1,57 @@
+import config from "../config/config";
```

Comment:
```suggestion
import config from "../config";
```
네이밍이 중복됩니다. 파일명을 index로 변경하는게 좋아보여요

Reply:
반영했습니다!